### DISCUSSION TITLE
HashiCorp adopting BUSL license

### DISCUSSION THREAD

**Discussion by to266** [kbst/terraform-kubestack#316] [General]
The [original announcement](https://www.hashicorp.com/blog/hashicorp-adopts-business-source-license) and [follow-up faq](https://www.hashicorp.com/license-faq#aug-10-announcement) for reference.

Are kubestack users or the kubestack itself impacted by this?

Also there seems to be https://opentf.org/ effort, at least starting. What are your thoughts @pst on all of this?

**Comment by pst** [comment]
It is my understanding that Hashicorp claims there is no change for end users. I don't see immediate impact for Kubestack, because Kubestack is not a cloud service and its users run Terraform themselves.

I have however underwritten the OpenTF effort and am committed to migrate Kubestack to a fork should a fork become necessary. Personally, I hope Hashicorp may reconsider and potentially join forces with other big companies willing to maintain Terraform truly open source under a foundation.

But who knows what the future brings.

**Reply by to266** [reply]
Glad to hear, thanks!
